### DISCUSSION TITLE
How do I return these buttons?

### DISCUSSION THREAD

**Discussion by krek4747** [Open-Shell/Open-Shell-Menu#1719] [Q&A]
### Describe the bug

I accidentally turned them off from the file explorer menu and I am not sure how to return these, when it seems to be turned on in the classic explorer menu.
![Screenshot (319)](https://github.com/Open-Shell/Open-Shell-Menu/assets/139050379/7672b9ed-41b0-47b4-8a1c-ce0d7bf922ce)
![Screenshot 2023-09-30 011709](https://github.com/Open-Shell/Open-Shell-Menu/assets/139050379/a7560cbe-6f97-4707-959d-303b1fdcc4ac)

### Area of issue

Windows Explorer

### To reproduce

1. I dont know what I am supposed to write here

### Open-Shell version

4.4.190

### Windows version

Windows 11 22h2

**Comment by Confuciusofessex** [comment]
Could someone give us an answer to this one, pretty please? Today mine have also gone awol, too. (4.4.191)

**Comment by Confuciusofessex** [comment]
Hitting F5 repeatably will give the words back (File Edit View Tools). Then select View>Toolbars>Classic Explorer Bar, and the buttons come back. Open another explorer window and repeat. Open in new tab, you'll need to do it again, etc.
